arrow/src/main/java/org/apache/iceberg/arrow/vectorized/parquet/VectorizedPageIterator.java

```diff
@@ -46,7 +45,7 @@ public VectorizedPageIterator(
     this.setArrowValidityVector = setValidityVector;
   }
 
-  private ValuesAsBytesReader plainValuesReader = null;
+  private VectorizedValuesReader valuesReader = null;
```

Comment:
I have replaced plainValuesReader with generic values reader to support different encodings.